### ISSUE TITLE
Namespaces.UnusedUses.UnusedUse - does not track function uses

### ISSUE THREAD

**Issue by ohartl** [slevomat/coding-standard#234]
I am using a function from another Namespace and would like to import it, seems like `Namespaces.UnusedUses.UnusedUse` is lacking to acknowledge to usage. This could be a "bug" or a missing property.

```php
<?php

use function GuzzleHttp\json_decode;

class A {

    public function foo(string $jsonInput)
    {
        return json_decode($jsonInput);
    }

}
```

Expected: No error

Actual:
```
FILE: ../A.php
---------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
---------------------------------------------------------------------------------------------------------------------------------------
 3 | ERROR | [x] Type GuzzleHttp\json_decode is not used in this file. (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)
---------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
---------------------------------------------------------------------------------------------------------------------------------------
```

**Comment by kukulich** [comment]
https://xkcd.com/583/

See test: https://github.com/slevomat/coding-standard/commit/6076e216d10d79b6c79ed341091997dcdda0c329
Results: https://travis-ci.org/slevomat/coding-standard/builds/324754732 https://ci.appveyor.com/project/slevomat/coding-standard/build/1.0.353
